docs/page-objects/fields.rst

```diff
@@ -4,55 +4,16 @@
 Fields
 ======
 
-Background
-----------
+A field is a read-only property in a :ref:`page object class <page-objects>`
+that is decorated with :meth:`@field <web_poet.fields.field>`, is named after a
+key of the :ref:`item <items>` that the page object class returns, and returns
+the value for that item key based on :ref:`inputs <inputs>`.
```

Comment:
I found this paragraph a bit hard to read. I guess it starts with "..a read-only property in a page-objects
that is decorated with..", where it's not immediately clear if a property is decorated, or a page object is decorated, or if you should use Python's `@property` (which must be read-only), and decorate it with `@field`.

Would it be possible to re-phrase it? Using bullet points instead of many commas may also help.

Reply:
What about https://github.com/scrapinghub/web-poet/pull/193/commits/1b812682a06fcb899f373f80e414883cebd10d10?